Google Cloud Platform Resource Manager CCF Connector Standard Table

v-gudivya opened the pull request:
Required items, please complete
   
   Change(s):
   - Added new CCF based connector for Google Cloud Platform Resource Manager. Implemented Standard table functionality.
   
   Reason for Change(s):
   - New CCF Development and deprecated custom table.

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

[Google Cloud Platform Resource Manager Checklist
](https://microsoftapc-my.sharepoint.com/:x:/r/personal/v-gudivya_microsoft_com//_layouts/15/Doc2.aspx?action=edit&sourcedoc=%7B322b36ec-3f25-4503-a7f2-604248a2432f%7D&wdOrigin=TEAMS-MAGLEV.p2p_ns.rwc&wdExp=TEAMS-TREATMENT&wdhostclicktime=1750225500457&web=1)

![GCP RM Connection Status](https://github.com/user-attachments/assets/96cc6b9b-59d3-4e9e-8f5c-718c95fc694f)

![GCP RM Graph](https://github.com/user-attachments/assets/b90937d7-86bd-4dee-89d8-8d2009749431)

![GCP RM UI](https://github.com/user-attachments/assets/d4ad5d11-755a-4656-9304-b78130e7640e)

![GCP RM Logs](https://github.com/user-attachments/assets/096accae-142a-4d9e-aa80-a186c995fdbb)

![GCP RM ReadMe 1](https://github.com/user-attachments/assets/dab91a74-6255-4b89-ad6a-d0456df37702)
![GCP RM ReadMe 2](https://github.com/user-attachments/assets/27deec4b-846d-4722-87be-bbd494e224e4)
![GCP RM ReadMe 3](https://github.com/user-attachments/assets/ea2e83da-12f2-4a6a-8224-535e1975d780)

v-gudivya committed:
Updated Files

v-gudivya committed:
Merge branch 'master' into v-gudivya-GCPRM

v-gudivya committed:
Modified the Domain

v-gudivya committed:
Renamed CCP to CCF

v-gudivya committed:
Merge branch 'master' into v-gudivya-GCPRM

v-gudivya committed:
Merge branch 'master' into v-gudivya-GCPRM

manishkumar1991 reviewed (dismissed):
Approved

v-gudivya committed:
Added Terraform scripts and ReadMe file. Updated to standard table.

v-pmalreddy committed:
Update standardLogStreams.ps1

v-pmalreddy committed:
Update standardLogStreams.ps1

v-pmalreddy committed:
Update standardLogStreams.ps1

v-pmalreddy committed:
Update standardLogStreams.ps1

v-pmalreddy committed:
Merge branch 'master' into v-gudivya-GCPRM

Alekhya0824 commented on Tools/Create-Azure-Sentinel-Solution/common/standardLogStreams.ps1:
cdn line is existing line

manishkumar1991 reviewed (dismissed):
Approved

v-prasadboke committed:
Merge branch 'master' into v-gudivya-GCPRM

v-prasadboke committed:
Update GCP Resource Manager connector UI and docs

Renamed the connector to 'Google Cloud Platform Resource Manager (via Codeless Connector Framework) (Preview)' in UI and templates. Updated setup script and tutorial links to point to official Azure Sentinel resources for improved accuracy and maintainability.
